app/models/user.rb

```diff
@@ -6,4 +6,6 @@ class User < ApplicationRecord
   def three_recent_posts
     posts.order(created_at: :desc).limit(3)
   end
```

Comment:
- [ ] Great job so far @rudicarrilloypr   There are a few changes I will suggest you implement and you'll be on your way to the next Milestone :wink: 
- [ ] Great job adding most of the tests, however, I noticed that per the project requirements, you were asked to `Add unit specs for all of your models' methods and validations.` But currently, this `three_recent_posts` method in your `User Model` hasn't been tested. So kindly include tests for that method as well :wink: